src/commands.rs

```diff
@@ -843,6 +844,24 @@ implement_commands! {
             .arg("LIMIT").arg(offset).arg(count)
     }
 
+    /// This command is like `ZRANGE`, but stores the result in `dst` destination key.
+    fn zrangestore<DST: ToRedisArgs, K: ToRedisArgs, M: ToRedisArgs, MM: ToRedisArgs>
+            (dst: DST, src: K, min: M, max: MM, opts: ZRangeStoreOptions) {
+        cmd("ZRANGESTORE").arg(dst).arg(src).arg(min).arg(max).arg(opts)
+    }
+
+    /// This command is like `ZRANGE`, but stores the result in `dst` destination key.
+    fn zrangestore_byscore<DST: ToRedisArgs, K: ToRedisArgs, M: ToRedisArgs, MM: ToRedisArgs>
+            (dst: DST, src: K, min: M, max: MM, opts: ZRangeStoreOptions) {
```

Comment:
If we add another field that takes

```rust
enum ZRangeStoreOrdering {
    Default,
    ByScore,
    ByLex,
}
```
we only need the single `zrangestore` function.

Reply:
I took the long way to get here :sweat_smile:. This is better.